src/azure-cli/HISTORY.rst

```diff
@@ -2977,6 +2977,10 @@ Added Cluster commands and trusted_service_access_enabled parameter for Networkr
 * storage blob generate-sas: User delegation SAS token support with --as-user
 * storage container generate-sas: User delegation SAS token support with --as-user
 
+**Security**
+
+* Added new commands group `az security va` and sub groups `az security va scans` `az security va results` `az security va baseline`
+
```

Comment:
We don't need modify this file, the history changes will be extracted from PR titles and be added here automatically when we have a release.

Reply:
Deleted